birdhouse/optional-components/canarie-api-full-monitoring/config/cowbird/canarie_api_full_monitoring.py.template

```diff
@@ -0,0 +1,9 @@
+import copy
+
+# See notes in 'components/cowbird/docker-compose-extra.yml'
+# about the conditional loading of this config in 'proxy' service.
+cowbird_cfg = copy.deepcopy(SERVICES["Cowbird"]["monitoring"]["Cowbird"])
+cowbird_cfg["request"]["url"] = "https://${PAVICS_FQDN_PUBLIC}/cowbird/"
+SERVICES["Cowbird"]["monitoring"]["Cowbird-public"] = cowbird_cfg
```

Comment:
Should leave the `if "Cowbird" in SERVICES` pre-check since there is no guarantee that the user would not have overridden this config.